### DISCUSSION TITLE
Skylake ReBar Assistance

### DISCUSSION THREAD

**Discussion by Hexicon42** [xCuri0/ReBarUEFI#244] [Q&A]
Greetings, 

Assistance is requested, as far as I know I have bricked my motherboard. I attempted to Follow all the instructions wich everything went fine until I ran "ReBarState.exe" and set it to 32.

I would receive a black screen and attempted to reset the bios by pulling my battery as instructed in the help article. that was no help...
Flashed back to a clean bios: gained my screen back but now I get these errors in a random order:

driver irql_less_or_not_equal 
UNEXPECTED_KERNEL_MODE_TRAP
TRAP_CAUSE_UNKNOWN
KMODE_EXCEPTION_NOT_HANDLED

I have no problems with post and entering/ configuring the bios. Errors occur during windows loading screen or within 5 min of bootup and auto crashes the machine. I've gone as far as starting fresh and reformatted the nvme drive with a fresh copy of windows from my 11th gen Desktop and kept the bios vanilla. no difference, I still crash constantly with the same errors.

- I've used both the the SOIC8 & traditional in bios/usb flash to vanilla.
- its impossible to get windows to last long enough to run ReBarState.exe and set it to 0...

specifications:
cpu 6700k
motherboard- gigabyte z170 gaming 7
16 gigs g.skill ram
500 gig nvme drive

**Comment by xCuri0** [comment]
If you're on vanilla BIOS RebarState being enabled won't do anything because it requires the ReBarDxe module to function. It could be that dual BIOS recovery is restoring the old BIOS image.
